server/user/queries.ts

```diff
@@ -103,6 +103,7 @@ export const updateUser = (
 
 	return User.update(filteredValues, {
 		where: { id: inputValues.userId },
+		individualHooks: true,
 	}).then(() => {
 		if (req.user.fullName !== filteredValues.fullName) {
 			updateUserData(req.user.id);
```

Comment:
this is necessary in order for the `User.afterUpdate` hook to run.

by default, those hooks only run for `user.update`, not `User.update({ where: { ...}})`